tmail-backend/event-bus-redis/src/test/java/org/apache/james/events/RabbitMQAndRedisEventBusWithRedisTLSSentinelTest.java

```diff
@@ -38,6 +38,7 @@
 import reactor.rabbitmq.QueueSpecification;
 import reactor.rabbitmq.Sender;
 
+@Disabled("Redis Sentinel tests are heavy to run")
```

Comment:
Might even just delete them I think, we test redis TLS in James already?

Reply:
Disabled. Keeping it does not hurt IMO.